<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Focus-QA-4" name="com.focus.testcases.BinOptionsTest" tests="110" failures="8" timestamp="1 Mar 2021 13:22:49 GMT" time="4187.941" errors="6">
  <testcase name="checkLoginAfterEnablePickFrompartiallyFilledBinsFirstWithFarthestAndFromBottom" time="12.514" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFrompartiallyFilledBinsFirst" time="14.914" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkOpeningStockRegisterReportWithFarthestOption" time="135.270" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkOpeningStockRegisterReportWithFarthestOption(BinOptionsTest.java:542)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkOpeningStockRegisterReportWithFarthestOption -->
  <testcase name="checkEraseAllTransationForTwoDifferentBatchescannotBePlacedInSameBIN" time="15.361" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionAllocateIntoPartiallyWithClosest" time="16.545" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForAllocateOIntoPartiallyFilledBinsFirstWithClosestAndFromTop" time="14.861" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionPickFrompartiallyFilledBinsFirstWithFarthest" time="17.658" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableAllocateIntoPartiallyWithFarthestAndFromTop" time="12.357" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionItemsArePlacedInBinsBasedOnTheirCategory" time="19.838" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithithPickFrompartiallyFilledBinsFirstWithFarthest" time="35.519" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionTwoDifferentBatchescannotBePlacedInSameBIN" time="17.458" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithithPickFrompartiallyFilledBinsFirstWithFarthestAndFromBottom" time="39.656" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksAfterEnablePickFromPartiallyFilledBinsFirst" time="33.105" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForAllocateOIntoPartiallyFilledBinsFirstWithClosestAndFromBottom" time="14.555" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionPickFrompartiallyFilledBinsFirstWithFarthestAndFromTop" time="17.541" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckSavingUnderTwoCategorysC1AndC2UnderPointOFSale" time="23.309" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableAllocateIntoPartiallyWithClosestAndFromBottom" time="12.337" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckSavingProductWithCategories" time="61.498" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFrompartiallyFilledBinsFirstWithFarthestAndFromTop" time="15.432" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkItemQueryOfFIFOItemWithClosetOption" time="17.078" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionAllocateIntoPartiallyWithFarthestAndFromTop" time="17.093" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionAllocateIntoPartiallyWithClosestAndFromTop" time="16.746" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkStockBalancesByItemByBinsReportsWithCLosetOption" time="135.430" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkStockBalancesByItemByBinsReportsWithCLosetOption(BinOptionsTest.java:462)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockBalancesByItemByBinsReportsWithCLosetOption -->
  <testcase name="checkSavingSalesInvoiceVoucherWithpickFromPartiallyFilledFirst" time="32.965" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithPickFrompartiallyFilledBinsFirstWithClosestAndFromBottom" time="34.149" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksWithOptionCloset" time="25.106" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithProductA" time="26.004" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithBatchBinProductBeforeEnableOption" time="33.809" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksWithProductAByUsingAutoAllocate" time="32.605" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFrompartiallyFilledBinsFirstWithClosestAndFromBottom" time="15.234" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkStockBalanaceByBinsReportsWithFarthestOption" time="133.802" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkStockBalanaceByBinsReportsWithFarthestOption(BinOptionsTest.java:558)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockBalanaceByBinsReportsWithFarthestOption -->
  <testcase name="checkEnteringDateIntoSecoundRowInOpeningStocksAfterOptionEnable" time="62.724" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionPickFrompartiallyFilledBinsFirstWithFarthestAndFromBottom" time="17.185" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableAllocateIntoPartiallyWithFarthest" time="12.320" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithReservedQtyWithClickOnAutoAllocate" time="82.589" classname="com.focus.testcases.BinOptionsTest">
    <error type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for element to be clickable: Proxy element for: DefaultElementLocator &#039;By.xpath: //td[@id=&#039;id_transaction_entry_detail_workflow_col_1-1&#039;]//input&#039; (tried for 20 second(s) with 1000 milliseconds interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: Proxy element for: DefaultElementLocator 'By.xpath: //td[@id='id_transaction_entry_detail_workflow_col_1-1']//input' (tried for 20 second(s) with 1000 milliseconds interval)
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:303)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:271)
	at com.focus.Pages.BinOptionsPage.checkSavingSalesInvoiceVoucherWithReservedQtyWithClickOnAutoAllocate(BinOptionsPage.java:17767)
	at com.focus.testcases.BinOptionsTest.checkSavingSalesInvoiceVoucherWithReservedQtyWithClickOnAutoAllocate(BinOptionsTest.java:356)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//td[@id='id_transaction_entry_detail_workflow_col_1-1']//input"}
  (Session info: chrome=88.0.4324.190)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
System info: host: 'FOCUS-QA-4', ip: '192.168.4.54', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.190, chrome: {chromedriverVersion: 87.0.4280.20 (c99e81631faa0..., userDataDir: C:\Users\D.n\AppData\Local\...}, goog:chromeOptions: {debuggerAddress: localhost:53509}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: d83d48c471eb1218830e9cc499569bae
*** Element info: {Using=xpath, value=//td[@id='id_transaction_entry_detail_workflow_col_1-1']//input}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:317)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:419)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:309)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy15.isDisplayed(Unknown Source)
	at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:315)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$100(ExpectedConditions.java:44)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:301)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:298)
	at org.openqa.selenium.support.ui.ExpectedConditions$23.apply(ExpectedConditions.java:686)
	at org.openqa.selenium.support.ui.ExpectedConditions$23.apply(ExpectedConditions.java:682)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:248)
	... 26 more
]]>
    </error>
  </testcase> <!-- checkSavingSalesInvoiceVoucherWithReservedQtyWithClickOnAutoAllocate -->
  <testcase name="checkLoginForTwoDifferentBatchescannotBePlacedInSameBIN" time="12.396" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithithPickFrompartiallyFilledBinsFirstWithFarthestAndFromTop" time="37.173" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithFIFOBinProductWithAllowPartallyEnableOption" time="28.256" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnablePickFrompartiallyFilledBinsFirstWithFarthest" time="12.445" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkReservingFullInwardQtyInSalesOrder" time="228.396" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnableOptionsForProductAsBatchBin" time="63.970" classname="com.focus.testcases.BinOptionsTest">
    <error type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//i[@class=&#039;icon-font6 icon-new&#039;]&quot;}
  (Session info: chrome=88.0.4324.190)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.12.0&#039;, revision: &#039;7c6e0b3&#039;, time: &#039;2018-05-08T14:04:26.12Z&#039;
System info: host: &#039;FOCUS-QA-4&#039;, ip: &#039;192.168.4.54&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_111&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.190, chrome: {chromedriverVersion: 87.0.4280.20 (c99e81631faa0..., userDataDir: C:\Users\D.n\AppData\Local\...}, goog:chromeOptions: {debuggerAddress: localhost:53509}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: d83d48c471eb1218830e9cc499569bae
*** Element info: {Using=xpath, value=//i[@class=&#039;icon-font6 icon-new&#039;]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//i[@class='icon-font6 icon-new']"}
  (Session info: chrome=88.0.4324.190)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
System info: host: 'FOCUS-QA-4', ip: '192.168.4.54', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.190, chrome: {chromedriverVersion: 87.0.4280.20 (c99e81631faa0..., userDataDir: C:\Users\D.n\AppData\Local\...}, goog:chromeOptions: {debuggerAddress: localhost:53509}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: d83d48c471eb1218830e9cc499569bae
*** Element info: {Using=xpath, value=//i[@class='icon-font6 icon-new']}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:317)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:419)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:309)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy15.isDisplayed(Unknown Source)
	at com.focus.Pages.BinOptionsPage.CheckEnableOptionsForProductAsBatchBin(BinOptionsPage.java:15052)
	at com.focus.testcases.BinOptionsTest.CheckEnableOptionsForProductAsBatchBin(BinOptionsTest.java:159)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CheckEnableOptionsForProductAsBatchBin -->
  <testcase name="checkSavingVoucherInOpeningStocksNewWithPickFrompartiallyFilledBinsFirstWithFarthestAndFromTop" time="34.159" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLogout" time="2.904" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFrompartiallyFilledBinsFirstWithFarthest" time="15.181" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithFIFOWithClosestAndFromTop" time="24.035" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithithPickFrompartiallyFilledBinsFirstWithCLoset" time="37.284" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionTwoDifferentItemsCannotBePlacedInsameBIN" time="17.243" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithFIFOWithClosestAndFromBottom" time="23.941" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnablePickFrompartiallyFilledBinsFirstWithCLoset" time="12.329" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckSavingBinsWithLevels" time="47.971" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionAllocateIntoPartiallyWithFarthestAndFromBottom" time="16.555" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableAllocateIntoPartiallyWithFarthestAndFromBottom" time="13.097" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFromPartiallyFilledBins" time="14.572" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckSavingBinsWithCategories" time="32.408" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnablePickFrompartiallyFilledBinsFirstWithFarthestAndFromTop" time="12.418" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForAllocateOIntoPartiallyFilledBinsFirstWithFarthestAndFromTop" time="15.280" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckSavingBinsWithDistances" time="45.804" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="chekSalesInvoiceBinScreenValidationWithProductB" time="40.876" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForAllocateIntoPartiallyWithClosest" time="15.017" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkOpeningStockRegisterReportWithClosetOption" time="135.279" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkOpeningStockRegisterReportWithClosetOption(BinOptionsTest.java:437)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkOpeningStockRegisterReportWithClosetOption -->
  <testcase name="CheckEnabletheOptionAllocateIntoPartiallyWithWithBinFirst" time="16.211" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionPickFrompartiallyFilledBinsFirstWithClosestAndFromTop" time="17.217" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionAllocateIntoPartiallyWithClosestAndFromBottom" time="16.910" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableAllocateIntoPartiallyWithClosest" time="12.748" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionPickFrompartiallyFilledBinsFirstWithCLoset" time="16.916" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithFIFOWithFarthest" time="22.483" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithFarthestOption" time="37.328" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithithPickFrompartiallyFilledBinsFirstWithClosestAndFromBottom" time="37.486" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionAllocateIntoPartiallyWithFarthest" time="16.916" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnablePickFrompartiallyFilledBinsFirstWithClosestAndFromBottom" time="12.467" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkStockLedgerReportByClosetOption" time="137.792" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkStockLedgerReportByClosetOption(BinOptionsTest.java:429)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockLedgerReportByClosetOption -->
  <testcase name="CheckEnabletheOptionPickFrompartiallyFilledBinsFirstWithClosestAndFromBottom" time="17.092" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAndRestoreOptionsCompanyAndLogin" time="116.828" classname="com.focus.testcases.BinOptionsTest">
    <error type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for element to be clickable: [[ChromeDriver: chrome on WINDOWS (d83d48c471eb1218830e9cc499569bae)] -&gt; id: btnSignin] (tried for 20 second(s) with 1000 milliseconds interval)
Build info: version: &#039;3.12.0&#039;, revision: &#039;7c6e0b3&#039;, time: &#039;2018-05-08T14:04:26.12Z&#039;
System info: host: &#039;FOCUS-QA-4&#039;, ip: &#039;192.168.4.54&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_111&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: [[ChromeDriver: chrome on WINDOWS (d83d48c471eb1218830e9cc499569bae)] -> id: btnSignin] (tried for 20 second(s) with 1000 milliseconds interval)
Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
System info: host: 'FOCUS-QA-4', ip: '192.168.4.54', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:303)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:271)
	at com.focus.Pages.LoginPage.clickOnSignInBtn(LoginPage.java:949)
	at com.focus.Pages.BinOptionsPage.checkLoginAndRestoreOptionsCompanyAndLogin(BinOptionsPage.java:13131)
	at com.focus.testcases.BinOptionsTest.checkLoginAndRestoreOptionsCompanyAndLogin(BinOptionsTest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- checkLoginAndRestoreOptionsCompanyAndLogin -->
  <testcase name="checkSavingSalesInvoiceVoucherWithithPickFrompartiallyFilledBinsFirstWithClosestAndFromTop" time="37.250" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckSavingBinsWithLevelsWithClosestAndFromBottom" time="47.932" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableTwoDifferentBatchescannotBePlacedInSameBIN" time="12.367" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkStockDetailsByBinsReportsWithClosetOption" time="94.424" classname="com.focus.testcases.BinOptionsTest">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementListHandler.invoke(LocatingElementListHandler.java:39)
	at com.sun.proxy.$Proxy16.get(Unknown Source)
	at com.focus.Pages.BinOptionsPage.checkStockDetailsByBinsReportsWithClosetOption(BinOptionsPage.java:19364)
	at com.focus.testcases.BinOptionsTest.checkStockDetailsByBinsReportsWithClosetOption(BinOptionsTest.java:445)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- checkStockDetailsByBinsReportsWithClosetOption -->
  <testcase name="checkStockBalancesByItemByBinsReportsWithFarthestOption" time="134.741" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkStockBalancesByItemByBinsReportsWithFarthestOption(BinOptionsTest.java:567)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockBalancesByItemByBinsReportsWithFarthestOption -->
  <testcase name="checkSavingSalesInvoiceVoucherWithClosetOption" time="42.293" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithPickFrompartiallyFilledBinsFirstWithClosestAndFromTop" time="33.908" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnablePickFromPartiallyFilledBinsFirst" time="12.350" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkStockLedgerReportByFarthestOption" time="143.106" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkStockLedgerReportByFarthestOption(BinOptionsTest.java:534)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockLedgerReportByFarthestOption -->
  <testcase name="checkSavingVoucher2InOpeningStocksNewWithFIFOWithFarthest" time="25.739" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithFIFOWithFarthestAndFromBottom" time="24.632" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFrompartiallyFilledBinsFirstWithClosestAndFromTop" time="15.357" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithProductB" time="12.790" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucher2InOpeningStocksAfterEnablePickFromPartiallyFilledBinsFirst" time="15.990" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableOptions" time="12.357" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingSalesInvoiceVoucherWithReservedQtyFromBin1" time="134.281" classname="com.focus.testcases.BinOptionsTest">
    <error type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for element to be clickable: Proxy element for: DefaultElementLocator &#039;By.xpath: //td[@id=&#039;id_transaction_entry_detail_workflow_col_1-1&#039;]//input&#039; (tried for 20 second(s) with 1000 milliseconds interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: Proxy element for: DefaultElementLocator 'By.xpath: //td[@id='id_transaction_entry_detail_workflow_col_1-1']//input' (tried for 20 second(s) with 1000 milliseconds interval)
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:303)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:271)
	at com.focus.Pages.BinOptionsPage.checkSavingSalesInvoiceVoucherWithReservedQtyFromBin1(BinOptionsPage.java:17419)
	at com.focus.testcases.BinOptionsTest.checkSavingSalesInvoiceVoucherWithReservedQtyFromBin1(BinOptionsTest.java:348)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//td[@id='id_transaction_entry_detail_workflow_col_1-1']//input"}
  (Session info: chrome=88.0.4324.190)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
System info: host: 'FOCUS-QA-4', ip: '192.168.4.54', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.190, chrome: {chromedriverVersion: 87.0.4280.20 (c99e81631faa0..., userDataDir: C:\Users\D.n\AppData\Local\...}, goog:chromeOptions: {debuggerAddress: localhost:53509}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: d83d48c471eb1218830e9cc499569bae
*** Element info: {Using=xpath, value=//td[@id='id_transaction_entry_detail_workflow_col_1-1']//input}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:317)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:419)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:309)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy15.isDisplayed(Unknown Source)
	at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:315)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$100(ExpectedConditions.java:44)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:301)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:298)
	at org.openqa.selenium.support.ui.ExpectedConditions$23.apply(ExpectedConditions.java:686)
	at org.openqa.selenium.support.ui.ExpectedConditions$23.apply(ExpectedConditions.java:682)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:248)
	... 26 more
]]>
    </error>
  </testcase> <!-- checkSavingSalesInvoiceVoucherWithReservedQtyFromBin1 -->
  <testcase name="checkEraseAllTransationForAllocateIntoPartiallyFilledBinsFirst" time="14.501" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForAllocateOIntoPartiallyFilledBinsFirstWithFarthest" time="15.226" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableAllocateIntoPartiallyWithWithBinFirst" time="12.332" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckSavingWarehousesWithBinsAllocation" time="34.026" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFromPartiallyFilledBinsFirst" time="14.608" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithAutoAllocateOption" time="16.917" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForPickFrompartiallyFilledBinsFirstWithFarthestAndFromBottom" time="15.349" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithPickFrompartiallyFilledBinsFirstWithCLoset" time="33.441" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkEraseAllTransationForAllocateOIntoPartiallyFilledBinsFirstWithFarthestAndFromBottom" time="15.098" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkStockDetailsByBinsReportsWithFarthestOption" time="134.115" classname="com.focus.testcases.BinOptionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.BinOptionsTest.checkStockDetailsByBinsReportsWithFarthestOption(BinOptionsTest.java:550)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockDetailsByBinsReportsWithFarthestOption -->
  <testcase name="chekSalesInvoiceBinScreenValidationWithProductA" time="50.314" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithFIFOWithFarthestAndFromTop" time="24.984" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithPickFrompartiallyFilledBinsFirstWithFarthest" time="23.662" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="CheckEnabletheOptionPickFromPartiallyFilledBinsFirst" time="15.968" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnablePickFrompartiallyFilledBinsFirstWithClosestAndFromTop" time="12.555" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkStockBalanaceByBinsReportsWithCLosetOption" time="93.633" classname="com.focus.testcases.BinOptionsTest">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementListHandler.invoke(LocatingElementListHandler.java:39)
	at com.sun.proxy.$Proxy16.get(Unknown Source)
	at com.focus.Pages.BinOptionsPage.checkStockBalanaceByBinsReportsWithCLosetOption(BinOptionsPage.java:19486)
	at com.focus.testcases.BinOptionsTest.checkStockBalanaceByBinsReportsWithCLosetOption(BinOptionsTest.java:453)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- checkStockBalanaceByBinsReportsWithCLosetOption -->
  <testcase name="checkEditingInBinsByIncreasingCapacity" time="122.246" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucher4InOpeningStocksAfterEnablePickFromPartiallyFilledBinsFirst" time="17.956" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucher3InOpeningStocksAfterEnablePickFromPartiallyFilledBinsFirst" time="33.076" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkLoginAfterEnableAllocateIntoPartiallyWithClosestAndFromTop" time="12.366" classname="com.focus.testcases.BinOptionsTest"/>
  <testcase name="checkSavingVoucherInOpeningStocksNewWithPickFrompartiallyFilledBinsFirstWithFarthestAndFromBottom" time="34.101" classname="com.focus.testcases.BinOptionsTest"/>
</testsuite> <!-- com.focus.testcases.BinOptionsTest -->
